docs/commercial/sharing.rst

```diff
@@ -66,3 +66,11 @@ For example:
 .. prompt:: bash $
    
    curl -H "Authorization: Token 19okmz5k0i6yk17jp70jlnv91v" https://docs.example.com/en/latest/example.html
+
+
+You can also use basic authorization, with the token as user and an empty password.
```

Comment:
Wouldn't be better to have a sub-section here: "Basic authorization" or similar? Also, maybe we can add a subsection immediately after the title saying "Token authorization", like:

* HTTP Authorization Header
  * Token authorization
  * Basic authorization

I feel it will be clearer.

Reply:
I like the idea of subsections, that way we can link to basic authorization directly. I'll update it